picopass/scenes/picopass_scene_key_menu.c

```diff
@@ -9,6 +9,39 @@ enum SubmenuIndex {
     SubmenuIndexWriteCustom,
 };
 
+typedef struct {
```

Comment:
This is an example of code I like, but feels unrelated and could probably go in beforehand (or after, which ever is easier)